app/resources/forms.js.template

```diff
@@ -156,6 +156,17 @@ function remove_profile_entry(profile_entry_index) {
   show($('add_profile_entry'));
 }
 
+// Show photo remove icon
+function show_photo_remove(photo_id) {
```

Comment:
Optional: show_photo_remove_button or show_photo_remove_icon would be more clear.